third_party/amd/lib/TritonAMDGPUToLLVM/DotOpToLLVM/WMMA.cpp

```diff
@@ -33,15 +33,17 @@ using ::mlir::triton::gpu::DotOperandEncodingAttr;
 
 enum class WMMAInstrType : uint8_t {
   // D = AB + C;
-  // typeof(D) == typeof(C)
-  // typeof(A) == typeof(B)
-  // typeof(D), typeof(A):
-  FP32_FP16,
-  FP32_BF16,
-  FP16_FP16,
-  BF16_BF16,
-  INT32_IU8,
-  INT32_IU4,
+  // typeof(A), typeof(B), typeof(C, D):
+  FP16_FP16_FP32,
```

Comment:
I think typically we use `<output>_<input>` ordering? At least in various AMD docs. Is there a reason we revert the that here?